contracts/artifacts/BatchExchange.json

```diff
@@ -1 +1 @@
entify touched order","owners":"array of addresses corresponding to touched orders","prices":"list of prices for touched tokens indexed by next parameter","tokenIdsForPrice":"price[i] is the price for the token with tokenID tokenIdsForPrice[i]"},"return":"the computed objective value of the solution     * Requirements: - Solutions for this `batchId` are currently being accepted. - Claimed objetive value is a great enough improvement on the current winning solution - Fee Token price is non-zero - `tokenIdsForPrice` is sorted. - Number of touched orders does not exceed `MAX_TOUCHED_ORDERS`. - Each touched order is valid at current `batchId`. - Each touched order's `executedSellAmount` does not exceed its remaining amount. - Limit Price of each touched order is respected. - Solution's objective evaluation must be positive.     * Sub Requirements: Those nested within other functions - checkAndOverrideObjectiveValue; Objetive value is a great enough improvement on the current winning solution - checkTokenConservation; for all, non-fee, tokens total amount sold == total amount bought"},"tokenAddressToIdMap(address)":{"details":"View returning ID of listed tokens","params":{"addr":"address of listed token."},"return":"tokenId as stored within the contract."},"tokenIdToAddressMap(uint16)":{"details":"View returning address of listed token by ID","params":{"id":"tokenId as stored, via BiMap, within the contract."},"return":"address of (listed) token"},"withdraw(address,address)":{"details":"Claims pending withdraw - can be called on behalf of others","params":{"token":"address of token to be withdrawn","user":"address of user who withdraw is being claimed.     * Emits an {Withdraw} event stating that `user` withdrew `amount` of `token`     * Requirements: - withdraw was requested in previous epoch - token was received from exchange in current auction batch"}}},"title":"BatchExchange - A decentralized exchange for any ERC20 token as a multi-token batch auction with uniform clearing prices. For more information visit: <https://github.com/gnosis/dex-contracts>"},"networks":{"1":{"address":"0x6F400810b62df8E13fded51bE75fF5393eaa841F","links":{"IdToAddressBiMap":"0xED4d05496C71e71cC2A8726af1242C22108d1761","IterableAppendOnlySet":"0xCDDB32b6Bb2808D5B5115dAab207479cE98d2636"},"transactionHash":"0xe74e8e965afc67f96a38caee794f008be33a7fe7ea96b9baad7b2963250ac36a"},"100":{"address":"0x66b89680A34394Cd73A4964AF60f56c9669C2452","links":{"IdToAddressBiMap":"0x048E53A455a058462eA58442E1D94Fbc955495cB","IterableAppendOnlySet":"0x57E6B987c2ccd421859A244dd22a0D5a62D88f91"},"transactionHash":"0x61c641256f7ca31f95f2e8a96e4a641758ceff3213905baf35efe5cc5a73256f"},"4":{"address":"0xC576eA7bd102F7E476368a5E98FA455d1Ea34dE2","links":{"IdToAddressBiMap":"0x5c4C6bf91240A5fdBfB9a1BEd8d43227046e2feA","IterableAppendOnlySet":"0x0D47D0548FDAD66B06E81a826EED8c687aCddBCB"},"transactionHash":"0xc5bf4f48747093a79b623c2a2392ccbdb73a3d788e7d1f0f53f640ea18496d90"}},"userdoc":{"methods":{}}}
```

Comment:
Was this updated with the contract `vendor` script? https://github.com/gnosis/dex-services/tree/master/contracts#vendor-script

Reply:
Yes , I will revert this. I was pulling in a new dex-contract version that had the updated addresses.